interface/billing/sl_receipts_report.php

```diff
@@ -540,43 +575,43 @@ function sel_diagnosis() {
 
  <tr>
   <td class="detail">
-   <?php echo $docnameleft; $docnameleft = "&nbsp;" ?>
+   <?php echo text($docnameleft); $docnameleft = " " ?>
   </td>
   <td class="detail">
    <?php echo oeFormatShortDate($row['transdate']) ?>
   </td>
 <?php if ($form_procedures) { ?>
   <td class="detail">
-   <?php echo empty($row['irnumber']) ? $row['invnumber'] : $row['irnumber']; ?>
+   <?php echo empty($row['irnumber']) ? text($row['invnumber']) : text($row['irnumber']); ?>
   </td>
 <?php } ?>
 <?php
         if ($form_proc_code && $form_proc_codetype) {
           echo "  <td class='detail' align='right'>";
             list($patient_id, $encounter_id) = explode(".", $row['invnumber']);
             $tmp = sqlQuery("SELECT SUM(fee) AS sum FROM billing WHERE " .
-              "pid = '$patient_id' AND encounter = '$encounter_id' AND " .
-              "code_type = '$form_proc_codetype' AND code = '$form_proc_code' AND activity = 1");
+              "pid = ? AND encounter = ? AND " .
+              "code_type = ? AND code = ? AND activity = 1", array($patient_id,$encounter_id,form_proc_codetype,$form_proc_code));
```

Comment:
Ensure you test everything that is modified(ie. the entire script in this case :) ). If you do the above line will likely throw an error :)
(hint: a $ is missing)